src/pages/partners/affiliate-ib/_partner-cta.tsx

```diff
@@ -59,9 +48,7 @@ const PartnerCTA = () => {
                     <Text align="center" color="white" mt="0.8rem">
                         {localize('The Deriv affiliate management team is here to help.')}
                     </Text>
-                    <LightButton external to="mailto:affiliates@deriv.com" tertiary is_mail_link>
-                        {localize('Contact us')}
-                    </LightButton>
+                    <LiveChatLinkText text="Contact us via live chat" weight="bold" />
```

Comment:
```suggestion
                    <LiveChatLinkText text="_t_Contact us via live chat_t_" weight="bold" />
```
please update this as well. :pray: